pallets/liquidity-proxy/src/lib.rs

```diff
@@ -34,10 +34,12 @@
 
 extern crate core;
 
-mod liquidity_aggregator;
+pub mod liquidity_aggregator;
 #[cfg(test)]
 mod mock;
 #[cfg(test)]
+mod new_tests;
```

Comment:
`new_tests` name is not descriptive, may be move the test to `tests` or give a better name?

Reply:
It will be done in #750